vg/vgimg/example_vgimg_test.go

```diff
@@ -0,0 +1,70 @@
+// Copyright ©2020 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package vgimg_test
+
+import (
+	"image"
+	"image/color"
+	"log"
+
+	"gonum.org/v1/plot"
+	"gonum.org/v1/plot/vg"
+	"gonum.org/v1/plot/vg/draw"
+	"gonum.org/v1/plot/vg/vgimg"
+)
+
+func ExampleUseDPI() {
+	p, err := plot.New()
+	if err != nil {
+		log.Fatalf("%+v", err)
+	}
+	p.Title.Text = "Title"
+	p.X.Label.Text = "X"
+	p.Y.Label.Text = "Y"
+
+	img := vgimg.NewWith(
```

Comment:
I'd suggest comments above the `vgimg.NewWith` calls for each of these examples explaining what the goal it.